src/engine/sim_engine/filesystem.rs

```diff
@@ -43,6 +43,10 @@ impl Filesystem for SimFilesystem {
         self.created
     }
 
+    fn path_to_mount_filesystem(&self, pool_name: &str, fs_name: &str) -> PathBuf {
+        vec!["/somepath", pool_name, fs_name].iter().collect()
```

Comment:
I'm not sure if this line was the root cause, but I saw this test failure in stratisd-nonrust:
```
tests/dbus/filesystem/test_properties.py::SetNameTestCase::testProps FAILED [  9%]
...
>       self.assertEqual(devnode, "/stratis/deadpool/fs")
E       AssertionError: dbus.String('/somepath/deadpool/fs', variant_level=1) != '/stratis/deadpool/fs'

tests/dbus/filesystem/test_properties.py:83: AssertionError
```

Reply:
Yes, I'll fix this.